astropy/io/fits/tests/test_header.py

```diff
@@ -951,7 +951,7 @@ def test_header_set_value_to_none(self):
 
         # Create a header that contains an undefined value and a defined
         # value.
-        hstr = "UNDEF   = \nDEFINED = 42"
+        hstr = "UNDEF   = \undefined = 42"
```

Comment:
```suggestion
        hstr = "UNDEF   = \nDEFINED = 42"
```
misguided typo fix?

Reply:
Yikes, how do we exclude this from future automated checks?

Reply:
Yeah, yikes. Codespell got updated. Got to add to our codespell ignore config in pyproject.toml